docs/sandbox.md

```diff
@@ -14,7 +14,7 @@ If you want maximum guardrails for a trusted repo, switch back to Read Only from
 
 - Every session starts in a sandbox. Until a repo is trusted, Codex enforces read-only access and will prompt before any write or command.
 - Marking a repo as trusted switches the default preset to Auto (`workspace-write` + `ask-for-approval on-request`) so Codex can keep iterating locally without nagging you.
-- The workspace always includes the current directory plus temporary directories like `/tmp`. Use `/status` to confirm the exact writable roots.
+- The workspace always includes the current directory plus temporary directories like `/tmp`.
```

Comment:
I feel like we need a `/status-dev` or some other command (omitted from the list in the product?) that is not as pretty as `/status` but dumps this sort of finer-grained information that is useful for debugging.